src/calz_unred.jl

```diff
@@ -1,16 +1,16 @@
 # This file is a part of AstroLib.jl. License is MIT "Expat".
 # Copyright (C) 2016 Mosè Giordano.
 
-function _calz_unred{T<:AbstractFloat}(wave::T, flux::T, ebv::T, r_v::T)
+function calz_unred(wave::T, flux::T, ebv::T, r_v::T) where {T<:AbstractFloat}
```

Comment:
Just slightly curious as to why calz_unred, sixty and nutate (and only them) use the new `Array{T} where T<:Integer` syntax.